tests/integration/test_models/test_full_battery_models/test_lithium_ion/test_dfn.py

```diff
@@ -21,6 +21,21 @@ def test_basic_processing(self):
         )
         modeltest.test_all()
 
+    def test_sensitivities(self):
+        options = {"thermal": "isothermal"}
+        model = pybamm.lithium_ion.DFN(options)
+        # use Ecker parameters for nonlinear diffusion
+        param = pybamm.ParameterValues(chemistry=pybamm.parameter_sets.Ecker2015)
+        var = pybamm.standard_spatial_vars
+        var_pts = {var.x_n: 10, var.x_s: 10, var.x_p: 10, var.r_n: 5, var.r_p: 5}
+        modeltest = tests.StandardModelTest(
+            model, parameter_values=param, var_pts=var_pts
+        )
+        modeltest.test_sensitivities(
+            #'Separator thickness [m]', 2e-05,
+            'Typical current [A]', 0.15652,
```

Comment:
can we test wrt "Current function [A]" instead of "Typical current [A]"? Typical current is a scaling parameter

Reply:
sure, I'll give "Current function" a go